### DISCUSSION TITLE
develop/how_to/filter_detections/

### DISCUSSION THREAD

**Discussion by giscus** [roboflow/supervision#1058] [Docs]
# develop/how_to/filter_detections/

A set of easy-to-use utilities that will come in handy in any computer vision project.

https://supervision.roboflow.com/develop/how_to/filter_detections/

**Comment by s-madhavan001** [comment]
how to activate/show in PolygonZone in video detection ?

**Reply by onuralpszr** [reply]
> how to activate/show in PolygonZone in video detection ?
> 

You can call in VideoSink callback to show it. Since you gonna use annotator same as others after you provide polygon and detections
